templates/app/httpd.yaml

```diff
@@ -312,12 +313,14 @@ objects:
             name: httpd-auth-configs
         containers:
         - name: httpd
+          securityContext:
+            privileged: true
```

Comment:
Why do we need `privileged` if we're switching to non-privileged ports?

Reply:
oops, wasn't supposed to commit this. Will fix in a bit.